src/python/WMCore/MicroService/MSPileup/MSPileup.py

```diff
@@ -116,7 +105,8 @@ def updatePileup(self, pdict):
         :return: results of MSPileup data layer (list of dicts)
         """
         authz_match(self.role, self.group)
-        return self.mgr.updatePileup(pdict)
+        # TODO: Implement logic for updating a pileup object and persist in the db
```

Comment:
the same argument here, I'm not sure it is correct change here.

Reply:
You are right, I didn't see it was actually a MSPileupData object (I got confused with the method names!).